src/nemo-connectivity/connectionhelper.cpp

```diff
@@ -134,6 +134,15 @@ ConnectionHelper::Status ConnectionHelper::status() const
     return m_status;
 }
 
+void ConnectionHelper::setSelectorVisible(bool selectorVisible)
+{
+    if (m_selectorVisible != selectorVisible)
```

Comment:
nitpick: opening brace should be on same line as if condition, according to Qt coding style.